tests/js/dictionary-tests.js

```diff
@@ -417,10 +433,13 @@ module.exports = {
         fields.removeListener(d)
 
         realm.write(() => { fields.field1=1 } )
-        TestCase.assertTrue(called,"Function c should be called")
-        fields.removeAllListeners();
+
         return new Promise((resolve, _) => {
             setTimeout(() => {
+                TestCase.assertEqual(called.a, 0, "Function a");
+                TestCase.assertEqual(called.b, 0, "Function b");
+                TestCase.assertEqual(called.c, 2, "Function c");
```

Comment:
Leave a comment explaining why 2 to have mercy on the next person to look at this.

Reply:
I'll add a comment